.github/workflows/tests_full.yml

```diff
@@ -0,0 +1,41 @@
+name: Tests Full
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+  pull_request:
+    types: [review_requested, ready_for_review, auto_merge_enabled]
```

Comment:
Can we please try to see if removing this event type will still allow workflow runs to be triggered manually?